index.html

```diff
@@ -2,15 +2,81 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Responsive Cake webpage</title>
+  <link rel="stylesheet" href="style.css" />
 </head>
 
 <body>
-    <!-- Add your markup here -->
+  <main>
+    <header>
+      <nav>
+        <div class="grid-one">
+          <img class="logo"
+            src="https://images.pexels.com/photos/1337380/pexels-photo-1337380.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500"
+            heigh="120" width="120" />
+          <div class="grid-one">
+            <h1>Contact</h1>
+          </div>
+        </div>
+        <section class="grid-two">
+          <div class="grid">
+            <h3>Home</h3>
+          </div>
+          <div class="grid">
+            <h3>Cakes</h3>
+          </div>
+          <div class="grid">
+            <h3>Ordering</h3>
+          </div>
+          <div class="grid">
+            <h3>Lessons</h3>
+          </div>
+          <div class="grid">
+            <h3>About</h3>
+          </div>
+        </section>
+      </nav>
+    </header>
+    <section class="general-hero-section">
+      <section class="first-section">
+        <div class="first-section-inf-div">
+          <div class="first-section-hello-div">
+            <p>HELLO</p>
+          </div>
+          <p>Information</p>
+        </div>
+      </section>
+
+      <section class="second-section">
+        <p>image carousel</p>
+      </section>
+    </section>
+
+    <section class="pictures-section">
+      <div class="each-picture">
+        <h3>Description</h3>
+      </div>
+      <div class="each-picture">
+        <h3>Description</h3>
+      </div>
+      <div class="each-picture">
+        <h3>Description</h3>
+      </div>
+      <div class="each-picture">
+        <h3>Description</h3>
+      </div>
+    </section>
+
+    <section>
```

Comment:
You don't need to wrap your footer in a section. `footer` is enough to tell the browser that this is a new section _and_ that it's a footer.
Also, move the footer outside of `main`. Your structure should be:
```
<header></header>
<main></main>
<footer></footer>
```